src/test/java/org/testeditor/fixture/host/integrationtest/HostDriverIntegrationTest.java

```diff
@@ -162,4 +163,62 @@ public void newTransactionTest() {
         hostDriverFixture.disconnect();
     }
 
+    @Test
+    public void readValueWithLocatorStrategyStartStopTest() {
+        assumeWindowsAndS3270Accessible();
+        // given
+        // hostDriverFixture in init
+        String user = System.getenv("USER");
+        String userPwd = System.getenv("USER_PWD");
+        String stopSystem = System.getenv("STOP_SYSTEM");
+        String valueToTest = System.getenv("VALUE_TO_TEST");
+        int testDefaultRow = Integer.parseInt(System.getenv("TEST_DEFAULT_ROW"));
+        int testDefaultColumn = Integer.parseInt(System.getenv("TEST_DEFAULT_COLUMN"));
+        int userPwdRow = Integer.parseInt(System.getenv("USER_PWD_ROW"));
+        int userPwdColumn = Integer.parseInt(System.getenv("USER_PWD_COLUMN"));
+
+        // when
+        hostDriverFixture.typeAt(user, standarRow, standardColumn);
+        hostDriverFixture.typeAt(userPwd, userPwdRow, userPwdColumn);
+        hostDriverFixture.sendControlCommand(ControlCommand.ENTER);
+        String valueAt = hostDriverFixture.readValueAt("2;1;2;12", LocatorStrategy.START_STOP);
+
+        // then
+        Assert.assertEquals(valueToTest, valueAt);
+
+        // stopping the host transaction
+        hostDriverFixture.typeAt(stopSystem, testDefaultRow, testDefaultColumn);
+        hostDriverFixture.sendControlCommand(ControlCommand.ENTER);
+        hostDriverFixture.disconnect();
+    }
+
+    @Test
+    public void readValueWithLocatorStrategyWidthTest() {
+        assumeWindowsAndS3270Accessible();
+        // given
+        // hostDriverFixture in init
+        String user = System.getenv("USER");
+        String userPwd = System.getenv("USER_PWD");
+        String stopSystem = System.getenv("STOP_SYSTEM");
+        String valueToTest = System.getenv("VALUE_TO_TEST");
+        int testDefaultRow = Integer.parseInt(System.getenv("TEST_DEFAULT_ROW"));
+        int testDefaultColumn = Integer.parseInt(System.getenv("TEST_DEFAULT_COLUMN"));
+        int userPwdRow = Integer.parseInt(System.getenv("USER_PWD_ROW"));
+        int userPwdColumn = Integer.parseInt(System.getenv("USER_PWD_COLUMN"));
```

Comment:
The setup and teardown stuff here is copy&pasted and should be extracted into a separate method.